### PR TITLE
Update cses-1074.mdx

### PR DESCRIPTION
**Pull request by TruaShamu**
Added java implementation to cses 1074 stick lengths

### CODE REVIEW

File: solutions/cses-1074.mdx [hunk 1]
```diff
@@ -48,3 +53,41 @@ int main(){
...
+		arr[i] = io.nextInt();
+	}
+	Arrays.sort(arr);
+
+	/**
+	 * Take the median stick length,
+	 * and calculate the cost to change each stick length to the median.
+	 */
+	long median = arr[N / 2];
+		long costs = 0;
```
Comment (dongliu0426) [inline solutions/cses-1074.mdx:81]:
```suggestion
	long costs = 0;
```

Comment (maggie-j-liu) [reply solutions/cses-1074.mdx:81]: it's not this part that has the wrong indentation, the entire main function's indentation is wrong

**Comment by TruaShamu** [comment]
I think it's fixed now.

File: solutions/cses-1074.mdx [hunk 2]
```diff
@@ -48,3 +53,41 @@ int main(){
...
+			arr[i] = io.nextInt();
+		}
+		Arrays.sort(arr);
+
+		/**
+		 * Take the median stick length,
+		 * and calculate the cost to change each stick length to the median.
+		 */
+		long median = arr[N / 2];
+	long costs = 0;
```
Comment (maggie-j-liu) [inline solutions/cses-1074.mdx:81]:
```suggestion
		long costs = 0;
```

**Review by maggie-j-liu — APPROVED** [review]

File: solutions/cses-1074.mdx [hunk 3]
```diff
@@ -48,3 +53,41 @@ int main(){
...
+		for (int i = 0; i < N; i++) {
+			arr[i] = io.nextInt();
+		}
+		Arrays.sort(arr);
+
+		/**
+		 * Take the median stick length,
+		 * and calculate the cost to change each stick length to the median.
+		 */
+		long median = arr[N / 2];
```
Comment (dongliu0426) [inline solutions/cses-1074.mdx:80]: can use `int` but doesn't really matter

**Review by dongliu0426 — APPROVED** [review]

File: solutions/cses-1074.mdx [hunk 4]
```diff
@@ -48,3 +53,41 @@ int main(){
...
+	public static void main(String[] args) {
+		Kattio io = new Kattio();
+		int N = io.nextInt();
+		int[] arr = new int[N];
+		for (int i = 0; i < N; i++) {
+			arr[i] = io.nextInt();
+		}
+		Arrays.sort(arr);
+
+		/**
```
Comment (SansPapyrus683) [inline solutions/cses-1074.mdx:76]:
```suggestion
		/*
```
2 stars are for doc comments only.

File: solutions/cses-1074.mdx [hunk 5]
```diff
@@ -48,3 +53,41 @@ int main(){
...
+</CPPSection>
+
+<JavaSection>
+
+```java
+
+import java.util.*;
+import java.io.*;
+
+public class stickLengths {
```
Comment (SansPapyrus683) [inline solutions/cses-1074.mdx:65]:
```suggestion
public class StickLengths {
```

File: solutions/cses-1074.mdx [hunk 6]
```diff
@@ -48,3 +53,41 @@ int main(){
 	return 0;
 }
 ```
+</CPPSection>
+
+<JavaSection>
+
+```java
+
```
Comment (SansPapyrus683) [inline solutions/cses-1074.mdx:61]:
```suggestion
```

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
very tiny stuff, other than that looks good

**Review by SansPapyrus683 — APPROVED** [review]

**Review by HarryWangATX — APPROVED** [review]

**Pull request merged**